tests/cypress/tests/e2e/mcpar/form.stepdef.js

```diff
@@ -100,24 +100,27 @@ When("I completely fill out a {string} form", (form) => {
   lastYear.setFullYear(today.getFullYear() - 1);
   const programName = "automated test - " + today.toISOString();
   cy.visit(`/${form.toLowerCase()}`);
-  cy.findByRole("button", { name: "Add managed care program" }).click();
-  cy.findByLabelText("Program name").type(programName);
+  cy.get("button").contains("Add managed care program").click();
+  cy.get('input[name="programName"').type(programName);
   cy.get('input[name="reportingPeriodStartDate"]').type(
     lastYear.toLocaleDateString("en-US")
   );
 
   cy.get('input[name="reportingPeriodEndDate"]').type(
     today.toLocaleDateString("en-US")
   );
-  cy.findByRole("checkbox").focus().click();
+  cy.get('input[name="combinedData"').check();
   cy.get("button[type=submit]").contains("Save").click();
 
   //Find our new program and open it
-  cy.findByText(programName)
-    .parent()
-    .find('button:contains("Edit")')
-    .focus()
-    .click();
+  cy.get("table").within(() => {
+    cy.get("td")
+      .contains(programName)
+      .parent()
+      .find('button:contains("Edit")')
+      .focus()
+      .click();
+  });
```

Comment:
findByText sometimes couldn't actually find things, and isn't actually a cypress command. So its been swapped to this.